libs/designer-ui/src/lib/chatbot/chatResources.ts

```diff
@@ -0,0 +1,163 @@
+export interface ChatResources {
```

Comment:
are we planning to be using all these props? We should avoid putting logic we're currently not using, cause this makes it harder to find during cleanup

Reply:
yeah i'll remove the ones i'm not using right now, and then add them in as needed